wordpress-mobile/WordPress-iOS
# Jetpack App: Updates the prologue screen

WPAuthenticator Related PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/592

Fixes #16362

### Video
https://user-images.githubusercontent.com/793774/116590322-9b3a4400-a8eb-11eb-8400-2a4a0baf5b24.mov

### Screenshots

| iPhone | iPad |
|:---:|:---:|
|<img width="487" alt="Screen Shot 2021-04-29 at 1 03 38 PM" src="https://user-images.githubusercontent.com/793774/116590097-60381080-a8eb-11eb-957c-4b321ce84d32.png">|<img width="836" alt="Screen Shot 2021-04-29 at 1 03 39 PM" src="https://user-images.githubusercontent.com/793774/116590164-6f1ec300-a8eb-11eb-9f30-23c296293a39.png">|

### To test:
1. Build the app using the Jetpack scheme
2. Launch the app
3. If you're logged in, log out
4. You'll see the new prologue screen
5. Tap around the view
6. Verify the view looks the same in dark mode and light mode
7. Follow the testing steps in the related PR

### Regression Notes
**1. Potential unintended areas of impact**
This includes change to WPAuthenticator and the NUXButton which impact other areas of the app and WooCommerce

**2. What I did to test those areas of impact (or what existing automated tests I relied on)**

- Manually tested on WP scheme
- Manually tested WooCommerce iOS
- Tested the other areas of the app that the NUXButton is used

**3. What automated tests I added (or what prevented me from doing so)**
None, this is a UI change

### PR submission checklist:
